packages/frontend/editor-ui/src/composables/useClipboard.ts

```diff
@@ -76,7 +79,9 @@ export function useClipboard(
 	return {
 		copy,
 		copied,
-		isSupported,
+		// When the `copy()` method is invoked from inside of the document picture-in-picture (PiP) window, it throws the error "Document is not focused".
+		// Therefore, we disable copying features in the PiP window for now.
+		isSupported: computed(() => isSupported && !isInPiPWindow.value),
```

Comment:
I didn't find any usages of `isSupported` in the codebase, so it should be safe to change `boolean` to `ComputedRef<boolean>` here.